scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/MockBigQuery.scala

```diff
@@ -123,6 +123,15 @@ class MockBigQuery private (private val bq: BigQuery) {
     queryResult(sqlQuery, flattenResults).map(bqt.fromTableRow)
   }
 
+  /** Run live DML statement against BigQuery service, substituting mocked tables with test data. */
+  def runDML(dmlStatement: String): TableReference = {
```

Comment:
I see we have a `isDML` in `QueryOps`. Maybe this is worth moving it to the `BigQuery` object with `scio` or `bigquery` package visibility  so we can make sure the input here is a DML with
```scala
require(isDML(dmlStatement), s"Expecting DML statement but got '$dmlStatement'")
```

Reply:
That makes sense to me! I've added this change.